cargo-pgrx/src/templates/lib_rs

```diff
@@ -23,10 +23,12 @@ mod tests {{
 /// It must be visible at the root of your extension crate.
 #[cfg(test)]
 pub mod pg_test {{
+    #[allow(clippy::needless_pass_by_value)]
     pub fn setup(_options: Vec<&str>) {{
```

Comment:
I would rather just fix the actual signature of `setup` for pgrx.

Reply:
i wasn't sure why you passed the ownership to the setup... but i guess this might be a relatively rare case, so might as well just pass `&[&str]`?

Reply:
P.S. Note that every single user of this code would need to update their signature - so it might not be worth the disruption?

Reply:
that is why we added a command to automatically upgrade code to pgrx.

Reply:
I also have merged quite a few PRs that will probably break a lot of extensions lately, and am going to merge another.

Reply:
something about actually spending my time on fixing critical soundness issues instead of fussing over lints.